batch/batch/driver/k8s_cache.py

```diff
@@ -1,74 +1,35 @@
-import time
-import asyncio
-import sortedcontainers
-
+from typing import Tuple
 from hailtop.utils import retry_transient_errors
 
+from hailtop.aiotools.time_limited_max_size_cache import TimeLimitedMaxSizeCache
+from ..batch_configuration import KUBERNETES_TIMEOUT_IN_SECONDS
+
 
 class K8sCache:
-    def __init__(self, client, refresh_time, max_size=100):
+    def __init__(self,
+                 client,
+                 refresh_time_ns: int = 5 * 1000 * 1000 * 1000,
```

Comment:
nit: can you put this in a constant called `FIVE_SECONDS_NS`? The math doesn't immediately happen in my head